srv/SetIO.srv

```diff
@@ -7,6 +7,9 @@
 #
 #  fun    The function to perform
 #  pin    Numeric ID of the pin to set
+#         Use pin 0-7 for digital / analog outputs
+#         Use pin 8-15 for configurable outputs
+#         Use pin 16, 17 for digital tool output
```

Comment:
You could also consider providing constants with names prefixed by the function.

Something like:

```yaml
PIN_DOUT0=0
PIN_DOUT1=1
...
PIN_DOUT7=7

PIN_AOUT0=0
...

PIN_CONF_OUT0=8
...

PIN_TOOL_DOUT0=16
PIN_TOOL_DOUT0=17
```

Users could then write something like:

```python
req = SetIORequest()
req.fun = SetIO.FUN_SET_DIGITAL_OUT
req.pin = SetIO.PIN_TOOL_DOUT0
req.state = SetIO.STATE_ON
...
```

Reply:
Uh, yes, will do that!

Reply:
Implemented in a991dbf